examples/llama2-7b.py

```diff
@@ -5,21 +5,21 @@
 
 # Add the parent directory to the path so we can import attribution
 sys.path.append(str(Path(__file__).parent.parent))
-from attribution.attribution import Attributor
+from attribution.local_attribution import LocalLLMAttributor
 
 model_id = "meta-llama/Llama-2-7b-chat-hf"
 model = AutoModelForCausalLM.from_pretrained(model_id, device_map="auto").cuda()
 tokenizer = AutoTokenizer.from_pretrained(model_id)
 embeddings = model.get_input_embeddings().weight.detach()
 
-attributor = Attributor(model=model, embeddings=embeddings, tokenizer=tokenizer)
-attr_scores, token_ids = attributor.get_attributions(
+attributor = LocalLLMAttributor(model=model, embeddings=embeddings, tokenizer=tokenizer)
+attr_scores, token_ids = attributor.compute_attributions(
     input_string="the five continents are asia, europe, afri",
     generation_length=7,
 )
 
 attributor.print_attributions(
-    word_list=tokenizer.convert_ids_to_tokens(token_ids),
+    word_list=tokenizer.convert_ids_to_tokens(token_ids),  # type: ignore
```

Comment:
The `# type: ignore` isn't a big deal obviously because it's an example. But I was curious so had a look.

It turns out there were two errors. The first was that `convert_ids_to_tokens` expects a list not a tensor (I don't know why my code didn't have this problem before) and so was returning the types `str | list[str] | Unknown`. If you pass it a list it returns `str | list[str]`.

The other issue is the `word_list` arg expecting a `list[str]`, so we can technically narrow the type with a runtime check using an assert or otherwise just converting it.

So overall, this works:
```python
word_list = tokenizer.convert_ids_to_tokens(token_ids.tolist())
word_list = [word_list] if isinstance(word_list, str) else word_list

attributor.print_attributions(
    word_list=word_list,
    attr_scores=attr_scores,
    token_ids=token_ids,
    generation_length=7,
)
```

Reply:
Sounds good!